app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1486,3 +1486,7 @@ $include-html-global-classes: $include-html-classes;
 // $include-legacy-visibility-classes: true;
 
 @import 'foundation';
+
+.button {
+  background-color: #01a1dd;
```

Comment:
Color literals like `#01a1dd` should only be used in variable declarations; they should be referred to via variable everywhere else.